File: p001/001.go
package main

import "fmt"

func main() {
	var arr [1000]int
	sum := 0

	for i := 0; i < 1000; i++ {
		arr[i] = i
	}

	for j := 0; j < len(arr); j++ {
		if arr[j]%3 == 0 || arr[j]%5 == 0 {
			sum += arr[j]
		}
	}

	fmt.Println(sum)
}

File: p001/001.js
/**
 * @author Alexi Jamal
 *
 * @description
 * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9.
 * The sum of these multiples is 23.
 * Find the sum of all the multiples of 3 or 5 below 1000.
 *
 */

let arr = [...Array(1000).keys()];
let sum = 0;

arr.forEach(number => {
  if (number % 3 == 0 || number % 5 == 0) sum += number;
});

console.log(sum);
//Execution time:
